addons/account/views/account_move_views.xml

```diff
@@ -976,12 +979,21 @@
                                            name="sale_info_group"
                                            attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund'))]}">
                                         <field name="invoice_user_id" domain="[('share', '=', False)]"/>
-                                        <field name="invoice_origin" force_save="1" string="Source Document"
+                                        <field name="ref" attrs="{'invisible': [('move_type', '!=', 'out_invoice')]}"/>
+                                        <field name="invoice_origin" string="Source Document"
```

Comment:
any reason the `force_save="1"` has been removed on this field?

Reply:
Nope :p